src/App.tsx

```diff
@@ -1,27 +1,70 @@
 import React from 'react';
 import './App.scss';
+import Clock from './Components/Clock/Clock';
+import { names } from './NamesForClock';
 
-const App: React.FC = () => {
-  const timerId: NodeJS.Timer = setInterval(() => {
-    const date: Date = new Date();
-
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  // eslint-disable-next-line
-  console.log(timerId);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
+type Props = {
+  clockName: string,
+  isClockVisible: boolean,
 };
 
+class App extends React.Component<{}, Props> {
+  state = {
+    clockName: 'React clock',
+    isClockVisible: true,
+  };
+
+  showClock = () => {
+    this.setState({
+      isClockVisible: true,
+    });
+  };
+
+  hideClock = () => {
+    this.setState({
+      isClockVisible: false,
+    });
+  };
+
+  setRandomName = () => {
+    this.setState({
+      clockName: names[Math.floor(Math.random() * names.length)],
+    });
+  };
+
+  render() {
+    const { isClockVisible, clockName } = this.state;
+
+    return (
+      <div className="App">
+        <h1 className="app-name">{clockName}</h1>
```

Comment:
If you have scss file, then I think you should rename classes with nesting: for example App__name. And use nesting in scss file.